docs/FAQ.md

```diff
@@ -144,3 +144,23 @@ TigerBeetle](https://github.com/tigerbeetle/tigerbeetle#quickstart),
 [follow our
 development](https://github.com/tigerbeetle/tigerbeetle#short-term-roadmap),
 and [give feedback](https://join.slack.com/t/tigerbeetle/shared_invite/zt-1gf3qnvkz-GwkosudMCM3KGbGiSu87RQ)!
+
+## Is TigerBeetle ACID-compliant?
+
+Yes. Let's discuss each part:
+
+1. Atomicity: TigerBeetle allows events within a batch to be
+   [linked](./reference/transfers.md#flagslinked). When an event fails
+   within a linked chain of events, all events are rolled back.
+
+2. Consistency: TigerBeetle's state machine ensures that accounts and
+   transfers remain consistent. When you create a transfer, associated
+   account credits and debits are guaranteed to be up-to-date.
```

Comment:
I'd like to work "invariant" and/or "double-entry" in here somehow. I think we can be more precise than "up-to-date".

:thinking: Something along the lines of "creating a transfer maintains all account invariants (e.g. balanced debits/credits)"? 

Also, maybe link to `./design/consistency.md`.

Reply:
> 2. Consistency: Accounts and transfers are immutable. Once created,
   users can no longer directly modify them. Creating a transfer
   maintains all account invariants (e.g. balanced
   debits/credits). See
   [./design/consistency.md](./design/consistency.md) for more
   details.

Any better? Feels sort of not to the point to me.

Reply:
Hmm, maybe I'm misremembering ACID :sweat_smile: 
I thought consistency was about maintaining database invariants.

I agree that the "users can no longer directly modify them" isn't super relevant though.
Maybe just your original text + the link at the end? Unless I'm misunderstanding and ACID consistency != `consistency.md`'s consistency, in which case we can leave it out.

Reply:
No I think you're right I just am not sure if everyone will connect account invariants to consistency without like an additional phrase or sentence.

Reply:
Great point re: not conflating with C in ACID.

What can help:

We typically speak of both “Financial Consistency” and “Raw Data Consistency” intentionally so as to contrast them and thereby show why double entry primitives are the right layer of abstraction (as opposed to the lower level general purpose DB primitives of SQL). For example, something like (needs work, but to give the idea): “the C in ACID provides raw data consistency across multiple row writes.... however, financial consistency is something more than this... [proceed to explain]"

Reply:
In other words, the argument is "at least ACID, and the best kind of ACID, but actually also more than ACID, not only ACID”

Reply:
> the C in ACID provides raw data consistency across multiple row reads/writes.... however, financial consistency is something more than this... [proceed to explain double entry invariants... such as money cannot be created nor destroyed, merely transferred from one account to another, with the transfer history immutable etc.]

Makes sense!

But I think in your discussion you didn't mention (or I didn't understand) what part of TigerBeetle actually guarantees raw data consistency?

Reply:
Ah or is it that "Strict Serializability", being how TigerBeetle is designed, is the answer to the Raw Data Consistency part?

Reply:
Pushed another version clarifying about strict serializability for traditional ACID guarantees at parts of your snippet for financial data guarantees.